pydal/_gae.py

```diff
@@ -83,3 +78,6 @@ def _validate(self, value):
                 return decimal.Decimal(value)
             raise TypeError("Property %s must be a Decimal or string."
                             % self._name)
```

Comment:
Mi concern is to have a lot of code into try ... except just to detect if we have GAE.